src/api/property/float.js

```diff
@@ -0,0 +1,6 @@
+export default {
+  default: 0,
+  serialize: String,
+  deserialize: parseFloat,
```

Comment:
For these (float, number, string) `deserialize` methods, should they behave the same as for boolean by returning `undefined` when they equal the default value so that the attribute is removed?